### PR TITLE
Remove title and timestamp from single image view, bounce photo icon

### PR DESCRIPTION
**Pull request by daehn**
# What's in this PR?

* From design review: The full screen image view should only show the close button.
* When saving from the image cells menu the camera icon should bounce.

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ImageMessageCell.m [hunk 1]
```diff
@@ -38,6 +38,8 @@
 #import "UIView+Borders.h"
 
 
+NSString * const ImageMessageCellDidSaveImageNotificationName = @"ImageMessageCellDidSaveImageNotificationName";
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ImageMessageCell.m]: This is unused?

Comment (daehn) [inline Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ImageMessageCell.m]: Yes

File: Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/Utility/SavableImage.swift [hunk 2]
```diff
@@ -21,28 +21,28 @@ import AssetsLibrary
 
 @objc final public class SavableImage: NSObject {
```
Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/Utility/SavableImage.swift]: I think it is more correct to say `saveable`

File: Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/Utility/SavableImage.swift [hunk 3]
```diff
@@ -21,28 +21,28 @@ import AssetsLibrary
...
+
+    init(data: NSData, orientation: UIImageOrientation) {
         imageData = data
         imageOrientation = orientation
-        saveCompletion = completion
         super.init()
     }
     
-    public func saveToLibrary() {
+    public func saveToLibrary(completion: ImageSaveCompletion?) {
```
Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/Utility/SavableImage.swift]: I think the selector name in Obj-C looks weird, would be cool to have something like `withCompletion:` postfix

File: Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/Utility/SavableImage.swift (hunk 2 above)
Comment (daehn) [inline Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/Utility/SavableImage.swift]: Apparently both can be used: https://en.wiktionary.org/wiki/saveable

Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/Utility/SavableImage.swift]: Ok, then fine!

**Comment by daehn** [comment]
@mikeger @typfel please re-review

**Pull request merged**